### PR TITLE
Create Cirrus_2020.2_GPU.md

### PR DESCRIPTION
**Pull request by jsindt**
Build instructions for GROMACS with GPUs

### CODE REVIEW

File: GROMACS/Cirrus_2020.2_GPU.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+module load mpt
+module load cmake
+
+
+2. Download & untar new edition of GROMACS.
+
+
+3. From the GROMACS directory, run the following.
+
+cmake ../ -DGMX_MPI=ON -DGMX_OPENMP=ON -DGMX_GPU=OFF -DGMX_X11=OFF \
```
Comment (aturner-epcc) [inline GROMACS/Cirrus_2020.2_GPU.md:15]: This cake command seems to set GPU  off and on. Looks like a typo

**Review by aturner-epcc — CHANGES_REQUESTED** [review]
Thanks @jsindt  looks like a typo in the cmake command. Can you check?

**Review by jsindt — COMMENTED** [review]
Good spot -- I've fixed the typo.

**Review by aturner-epcc — APPROVED** [review]
Thanks for fixing

**Pull request merged**